docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -1,116 +1,101 @@
+```bash title="Terminal Command"
+gsbquota
 ```
-### Disk
-
-Unlike personal home directories which have a 50 GB quota, faculty project directories on <a href="/storage/fileStorage.html" target="_blank">yens/ZFS</a> are much bigger (1T default).
-Disk storage is a finite resource, however, so to allow us to continue to provide large project spaces please always be aware of your disk footprint. This includes compressing files when you are able, and removing intermediate and/or temp files whenever possible.
-See the <a href="/storage/fileStorage.html" target="_blank">yen file storage page</a> for more information about file storage options.
-
-Disk quotas on all yen servers can be reviewed by using the ```gsbquota``` command. It produces output like this:
 
-```bash
-nrapstin@yen1:~$ gsbquota
-/home/users/nrapstin: currently using 39% (20G) of 50G available
+```{.yaml .no-copy title="Terminal Output"}
+currently using 39% (20G) of 50G available
 ```
 
 You can also check the size of your project space by passing in a full path to your project space to `gsbquota` command:
 
-```bash
-nrapstin@yen1:~$ gsbquota /zfs/projects/students/<my-project-dir>/
-/zfs/projects/students/<my-project-dir>/: currently using 39% (78G) of 200G available
+```bash title="Terminal Command"
+gsbquota /zfs/projects/students/<my-project-dir>/
 ```
 
+```{.yaml .no-copy title="Terminal Output"}
+/zfs/projects/students/<my-project-dir>/: currently using 39% (78G) of 200G available
+```
 
-## Example
-We are going to continue using the same R example, `investment-npv-parallel.R`, and experiment running it on multiple cores and monitoring our resource consumption.
-
-To monitor the resource usage while running a program, we will need three terminal windows that are all connected to the **same** yen server.
-
-Check what yen you are connected to in the first terminal:
+Unsure what's taking up space? The `gsbbrowser` command scans your home directory and provides a visual representation of your directories and files and their associated sizes. You can also provide a path to a project directory to scan that directory:
 
-```bash
-$ hostname
+```bash title="Terminal Command"
+gsbbrowser /zfs/projects/students/<my-project-dir>/
 ```
 
-Then `ssh` to the same yen in the second and third terminal windows. So if I am on `yen3`, I would open two new terminals and `ssh` to
-the `yen3` in both so I can monitor my resources when I start running the R program on `yen3`.
+## Example: Monitoring an R Script
+Here we present an example in R, to illustrate monitoring code running on multiple cores.
 
-```bash
-$ ssh yen3.stanford.edu
-```
+To monitor the resource usage while running a program, we will need *three* terminal windows that are all connected to the **same** Yen server.
 
-Once you have three terminal windows connected to the same yen, run the `investment-npv-parallel.R` program after loading the R module
-in one of the terminals:
+`ssh` to `yen3` in the first terminal window.
```

Comment:
It does not have to be yen3, they can just ssh to yens first then ssh to the same machine twice below.